scripts/pages.mjs

```diff
@@ -43,6 +43,69 @@ class ClassPages extends Application {
     html[0].querySelectorAll(".class-nav img[data-action]").forEach(n => {
       n.addEventListener("click", this._onClickClass.bind(this));
     });
+    this.navScroller(html);
+  }
+
+  /**
+   * Activates the scroll functionality for the class icons gallery.
+   * @param {JQuery} html The HTML content of the rendered ClassPages application.
+   */
+  navScroller(html) {
+    const gallery = html[0].querySelector(".class-nav .gallery");
+    let isScrolling = false;
+
+    gallery.addEventListener(
+      "wheel",
+      function (event) {
+        if (isScrolling) return;
+
+        isScrolling = true;
+        const direction = Math.sign(event.deltaY);
+
+        if (direction > 0) {
+          this.scrollToNextIcon(gallery);
+        } else if (direction < 0) {
+          this.scrollToPreviousIcon(gallery);
```

Comment:
These two nested scroll functions are small and not used outside of this function. They also use some similar logic. I think it would be fine to consolidate this code into a single function.